### ISSUE TITLE
!langspec(ocaml)(c)(python) qui se promène

### ISSUE THREAD

**Issue by vvrav** [MarcdeFalco/MarcdeFalco.github.io#3]
Si je ne dis pas de bêtise, cette ligne n'a rien à faire là (je ne sais pas si tu as des préférences quant à la manière de signaler d'éventuelles coquilles).

https://github.com/MarcdeFalco/MarcdeFalco.github.io/blob/433ace93cbd0c7e989f8f504d9c59cc89433e9aa/chap_algorithmique_analyse_algorithme.html#L708

**Comment by vvrav** [comment]
Aaah mais d'ailleurs à la réflexion, j'imagine que c'est un truc qui n'est pas passé comme prévu, vu qu'il semblerait que la suite soit aussi prévue avec un choix de langage.

![Capture d’écran du 2022-10-05 09-50-37](https://user-images.githubusercontent.com/47926132/194009047-e971368a-54c1-4287-91a1-671cd1f83dbd.png)

**Comment by MarcdeFalco** [comment]
Oui, c'est ça. Je suis en train de changer d'avis sur l'organisation du texte pour faire des documents pour les trois langages en même temps.
